networkx/generators/geometric.py

```diff
@@ -56,23 +58,12 @@ def random_geometric_graph(n, radius, dim=2, pos=None, metric=None):
         Dimension of graph
     pos : dict, optional
         A dictionary keyed by node with node positions as values.
-    metric : function
-        A metric on vectors of numbers (represented as lists or
-        tuples). This must be a function that accepts two lists (or
-        tuples) as input and yields a number as output. The function
-        must also satisfy the four requirements of a `metric`_.
-        Specifically, if *d* is the function and *x*, *y*,
-        and *z* are vectors in the graph, then *d* must satisfy
+    norm : float
+        Which Minkowski norm to use.  `p` has to meet the condition
```

Comment:
Since this must be a number greater than one, this should be the L_p metric, not the Minkowski metric. It would be nice to link to the definition of the L_p metric here.